ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/target/Messages.properties

```
@@ -42,6 +42,8 @@ LocalTargetHandle_2=Unable to generate memento for target platform
 LocalTargetHandle_3=Failed to delete target definition: {0}
 LocalTargetHandle_4=Error saving target definition {0}
 LocalTargetHandle_5=Error creating target file
+P2TargetUtils_cant_refresh_artifacts=Can't refresh artifact repository {0}
+P2TargetUtils_cant_refresh_metadata=Can't refresh metadata repository {0} reported metadata might be outdated
```

It doesn't read like a grammatically complete sentence:
```suggestion
P2TargetUtils_cant_refresh_metadata=Can't refresh metadata repository {0} so the reported metadata might be outdated
```